src/api/app/datatables/package_datatable.rb

```diff
@@ -0,0 +1,38 @@
+# NOTE: Folowing: https://github.com/jbox-web/ajax-datatables-rails#using-view-helpers
+class PackageDatatable < AjaxDatatablesRails::ActiveRecord
+  extend Forwardable
+
+  def_delegator :@view, :link_to
+  def_delegator :@view, :package_show_path
+  def_delegator :@view, :time_ago_in_words
+
+  def initialize(params, opts = {})
+    @project = opts[:project]
+    @view = opts[:view_context]
+    super
+  end
+
+  def view_columns
+    # Declare strings in this format: ModelName.column_name
+    # or in aliased_join_table.column_name format
+    @view_columns ||= {
+      name: { source: 'Package.name', cond: :like },
+      changed: { source: 'Package.updated_at', cond: :like }
+    }
+  end
+
+  # rubocop:disable Naming/AccessorMethodName
```

Comment:
I would exclude the datatables directory instead of doing it inline in all the files

Reply:
Disabling the whole file because we want to ignore a cop in one line, or a few, sounds like overkill to me.

Reply:
I mean to exclude `src/api/app/datatables` for `Naming/AccessorMethodName` in `.rubocop` (with a nice comment explaining why we are not going to finish this)

Reply:
Problem is that this would require me to convert all `Naming/AccessorMethodName` offenses we have right now and there are to many.

Reply:
I think this should be straightforward (or there is a bug in Rubocop). But this is not that important, do not block this PR for this. I will take a look into it and send a PR if I find a solution. Thanks for the branch.